pandas/tests/io/excel/test_readers.py

```diff
@@ -19,16 +19,27 @@
 
 
 @contextlib.contextmanager
-def ignore_xlrd_time_clock_warning():
+def ignore_engine_warnings():
     """
-    Context manager to ignore warnings raised by the xlrd library,
-    regarding the deprecation of `time.clock` in Python 3.7.
+    Context manager to ignore warnings raised by the excel engine that would
+    interfere with asserting warnings are reaised.
     """
     with warnings.catch_warnings():
+        # raised by the xlrd library, regarding the deprecation of `time.clock`
+        # in Python 3.7.
         warnings.filterwarnings(
             action='ignore',
             message='time.clock has been deprecated',
             category=DeprecationWarning)
+
+        # raised by the openpyxl library, if unsupported extensions to the
```

Comment:
I may be conflicting with something I told you before but I think these should be separate warning filters